mirror-maker-2/flexible/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - deployment.yaml
+
+# This configmap should be enabled in the concrete usages of this base
+#configMapGenerator:
```

Comment:
why is this commented out here, but not for the `active-passive-ssl` case?

Reply:
Because here it is just an example on how to be created in the concrete usages, as the comment says, and in active-passive-ssl we're creating the configmap with the template file